### DISCUSSION TITLE
Is it possible to pass the --override-k parameter ?

### DISCUSSION THREAD

**Discussion by refazul** [ericaltendorf/plotman#253] [General]
When i set ```k=25``` in the plotter section of the ```plotman.yaml``` and then run ```plotman interactive``` in the terminal, i get this error on the log file:
```
k=32 is the minimun size for farming.
If you are testing and you want to use smaller size please add the --override-k flag.
```
Where should i add this parameter ?

**Comment by ericaltendorf** [comment]
The only way to do this right now is to hack it into the source code where
`manager.py` kicks off the plot job.  We do aim to fix this, sorry about
the inconvenience.

**Reply by altendky** [reply]
The main hangup is taking the time to decide how to make sure people don't end up forgetting and plotting a pile of k25 plots because they forgot.  I expect making it a command line parameter when starting plotman and having some extra warning banner in interactive mode would suffice.

**Reply by refazul** [reply]
OK. I added '--override-k' on my local copy of [manager.py](https://github.com/ericaltendorf/plotman/blob/main/src/plotman/manager.py#L109) file and it worked as expected ! Many thanks !!
